### PR TITLE
Fixed CAT-1888, 1898, 1900

### PR DESCRIPTION
**Pull request by arabind**
@revinjacob : Please review and merge. This PR contains:
1- CAT-1888
2- CAT-1898
3- CAT-1900
4- Fixed docker image issue and logo issue in instances table view.

### CODE REVIEW

File: client/cat3/src/partials/sections/dashboard/workzone/azureARM/azureARM.html [hunk 1]
```diff
@@ -5,6 +5,9 @@
 			<span class="panel-title">
 				<span>Deployments</span>
 			</span>
+			<span class="marginleft15" title="Refresh" ng-click="refreshCurrentPage();" alt="Refresh View">
+				<i class="fa fa-refresh cursor"></i>
```
Comment (revinjacob) [inline client/cat3/src/partials/sections/dashboard/workzone/azureARM/azureARM.html]: Here this marginleft15 class can be avoided. Instead add a margin right to the tab name span class, here 'Deployments' (panel-title class)

File: client/cat3/src/partials/sections/dashboard/workzone/azureARM/azureARMCtrl.js [hunk 2]
```diff
@@ -24,28 +26,33 @@
 
 			$rootScope.$on('WZ_ENV_CHANGE_START', function(event, requestParams){
 				$scope.isAzureARMPageLoading = true;
+				$scope.setFirstPageView();
 				$scope.envParams=requestParams;
```
Comment (revinjacob) [inline client/cat3/src/partials/sections/dashboard/workzone/azureARM/azureARMCtrl.js]: The implementation of setFirstPageView method needs to be updated to 
$scope.currentCardPage = $scope.paginationParams.page = 1;

File: client/cat3/src/partials/sections/dashboard/workzone/container/containerCtrl.js [hunk 3]
```diff
@@ -69,6 +69,11 @@
 					$scope.errorMessage = "No Records found";
 				});
 			};
+
+			$scope.refreshCurrentPage = function(){
+				$scope.getContainerList();
+			};
+
 			$scope.stopDockerFunction = function(actionType){
```
Comment (revinjacob) [inline client/cat3/src/partials/sections/dashboard/workzone/container/containerCtrl.js]: SHOW_LATEST event type is missing in containers. This is needed to show newly launched containers

Comment (revinjacob) [inline client/cat3/src/partials/sections/dashboard/workzone/container/containerCtrl.js]: event and setFirstPage method needs to be added

File: client/cat3/src/partials/sections/dashboard/workzone/instance/instance.html [hunk 4]
```diff
@@ -151,7 +154,7 @@
 									</li>
 									<ul class="paddingleft32 instanceTaskListUL">
 										<li ng-show="inst.taskIds.length==0">No TaskLinks Available</li>
-										<li class="cursor" ng-repeat="taskDetail in inst.Ids">{{taskDetail.name}}</li>
```
Comment (revinjacob) [inline client/cat3/src/partials/sections/dashboard/workzone/instance/instance.html]: As the data display is based on inst.tasks array, the 'No TaskLinks Available' check should refer to the same array.

**Comment by revinjacob** [comment]
@arabind, please address review comments and resolve the conflicts caused due to merge of PR#261

**Comment by arabind** [comment]
@revinjacob : Please review and merge the code for 
1- Handled comments for PR#262
2- Fixed merge conflicts in instance.html with PR#261
3- Fixed CAT-1906

**Pull request merged**